src/room.ts

```diff
@@ -25,12 +25,12 @@ type RoomObj = {
   topic:      string
   ownerUin:   number
   memberList: Contact[]
-  nickMap:    Map<string, string>
-  remarkMap:  Map<string, string>
-  displayMap: Map<string, string>
+  nameMap:      Map<string, string>
+  aliasMap:     Map<string, string>
+  roomAliasMap: Map<string, string>
```

Comment:
If we use Contact.find() instead, then when use function `member()` to get Contact would be asynchronous， which means it will use 
`public async member(queryArg: MemberQueryFilter | string): Promise<Contact | null>` 
but actually we want to use synchronous function.

And, we must use alias(remark) in room, which I have said in #104 :

You said in [room.ts](https://github.com/wechaty/wechaty/blob/master/src/room.ts)

> ISSUE #104 never use remark name because sys group message will never use that

**But sys group message use remark.** 

For example, when the bot set remark for A, then A changed the topic, then sys message shows the remark, not roomAlias.

I have summarize in [#173](https://github.com/wechaty/wechaty/issues/173):

>

> Display order:
> 
> @ Event
> 
> When someone @ a contact in a room, wechat recognise the name order as follows:
> displayName > nickName
> 
> system message
> 
> When room event happens(join, leave, changetopic), system message recognise the name order as follows:
> remark > nickName
> 
> on-screen Names
> 
> The contact name showed to the wechat user in the group
> remark > displayName > nickName